src/core/picked_player.py

```diff
@@ -0,0 +1,18 @@
+class PickedPlayer:
+    def __init__(
+        self,
+        player_id: int,
+        player_name: str,
+        team_name: str,
+        team_abbreviation: str,
+        position: str,
+        position_abbreviation: str,
+        score: int,
```

Comment:
Do you think we need all of these parameters? I mean, to decouple what the API returns and what are we modeling, I would handle only the parameters we need and add new parameters when needed.

Reply:
These parameters are there to use it to models the output